docs/tables/aws_vpc_flow_log/index.md

```diff
@@ -140,15 +151,17 @@ partition "aws_vpc_flow_log" "my_logs_prefix" {
 }
 ```
 
-### Collect logs from local files
+### Exclude rejected traffic logs
 
-You can also collect VPC Flow Logs from local files.
+Use the filter argument in your partition to exclude rejected traffic logs and reduce the size of local log storage.
 
 ```hcl
-partition "aws_vpc_flow_log" "local_logs" {
-  source "file"  {
-    paths       = ["/Users/myuser/vpc_flow_logs"]
-    file_layout = `%{DATA}.json.gz`
+partition "aws_vpc_flow_log" "my_logs_instance" {
+  filter = "action <> 'REJECT'"
```

Comment:
Do we use `<>` or `!=` more commonly (I think the latter)?

Reply:
Yes, `!=` is the more commonly used operator. I've updated the query to use `!=` instead.